src/utils/is-sort-imports-ignored.ts

```diff
@@ -6,6 +6,6 @@ import { getAllCommentsFromNodes } from './get-all-comments-from-nodes';
 export const isSortImportsIgnored = (nodes: Statement[]) =>
     getAllCommentsFromNodes(nodes).some(
         (comment) =>
-            comment.loc.start.line === 1 &&
+            comment.loc?.start.line === 1 &&
```

Comment:
May be :wink
```suggestion
           comment.loc && comment.loc.start.line === 1 &&
```

Reply:
Fixed, but what a benefit to write in this way? It would never equal 1 if comment.loc was undefined.